docs/Makefile

```diff
@@ -3,6 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
+SPHINX_APIGEN = sphinx-apidoc
```

Comment:
When I tried to build the doc by `make html`, it complains about missing sphinxcontrib-mermaid. That can be added to the requirements.txt

Reply:
Stephen says this is fixed ... marking resolved.